### PR TITLE
fix: Remove stake sign logic from protocol

### PR DESCRIPTION
**Pull request by rafaelugolini**

### CODE REVIEW

File: libs/base/src/lib/components/wallet/PendingTransactions.tsx [hunk 1]
```diff
@@ -138,26 +140,32 @@ export const PendingTransaction: FC<{
...
   const { cancel, send } = useTransactionsDispatch()
   const { estimationError, gasLimit, gasPrice, insufficientBalance } = useGas()
+  const [{ appName }] = useBaseCtx()
 
   const [stakeSignatures, setStakeSignatures] = useStakeSignatures()
   const [isStakeSigned, setIsStakeSigned] = useState<boolean>(false)
   const [isStakeSignedForm, setIsStakeSignedForm] = useState<boolean>(false)
   const stakeSignedFunctions = ['compoundRewards', 'stake(uint256)']
 
+  const isGovernance = appName === APP_NAME.GOVERNANCE
```
Comment (doncesarts) [inline libs/base/src/lib/components/wallet/PendingTransactions.tsx:150]: is there a file for common functions ?  , the same logic is used in 2 different files

Comment (rafaelugolini) [reply libs/base/src/lib/components/wallet/PendingTransactions.tsx:150]: maybe we could put this in the `useBaseCtx`
@chrisjgf @JamesLefrere what do you think?

Comment (chrisjgf) [reply libs/base/src/lib/components/wallet/PendingTransactions.tsx:150]: Could do, not too fussed though. Constant is pulled out

File: libs/base/src/lib/context/AccountProvider.tsx [hunk 2]
```diff
@@ -257,19 +262,20 @@ const OnboardProvider: FC<{
...
-    fetch(`${API_ENDPOINT}/signature`)
-      .then(resp => resp.json())
-      .then(json => {
-        setStakeSignatures(prevSignatures => ({
-          ...prevSignatures,
-          message: json.message,
-        }))
-      })
-      .catch(console.error)
+    isGovernance &&
```
Comment (chrisjgf) [inline libs/base/src/lib/context/AccountProvider.tsx:265]:
```suggestion
    if (isGovernance) {
```
Might be nicer as a block here to improve readability & follow the pattern in the func

**Review by chrisjgf — APPROVED** [review]

**Pull request merged**